packages/calypso-e2e/src/lib/components/editor-nav-sidebar-component.ts

```diff
@@ -26,61 +26,23 @@ export class EditorNavSidebarComponent {
 		this.editor = editor;
 	}
 
-	/**
-	 * Opens the sidebar if not already open.
-	 */
-	async openSidebar(): Promise< void > {
-		if ( envVariables.VIEWPORT_NAME === 'mobile' ) {
-			return;
-		}
-
-		if ( await this.sidebarIsOpen() ) {
-			return;
-		}
-
-		const locator = this.editor.locator( selectors.sidebarButton );
-		await locator.click();
-	}
-
-	/**
-	 * Closes the sidebar if open.
-	 */
-	async closeSidebar(): Promise< void > {
-		if ( envVariables.VIEWPORT_NAME === 'mobile' ) {
-			return;
-		}
-
-		if ( ! ( await this.sidebarIsOpen() ) ) {
-			return;
-		}
-
```

Comment:
This has been moved to the Toolbar component. This is for two reasons...
1. It's technically a toolbar button
2. It allows to remove the mobile logic from this component so it can be shared across site and page editors.